src/Turing.jl

```diff
@@ -62,6 +63,16 @@ end
 (model::Model)(args...; kwargs...) = model.f(args..., model; kwargs...)
 function runmodel! end
 
+struct Selector
+    gid :: UUID
+end
+Selector() = Selector(uuid1())
+const DEFAULT_SELECTOR = Selector(UUID("00000000-0000-0000-0000-000000000000"))
+const INVALID_SELECTOR = Selector(UUID("00000000-0000-0000-0000-000000000001"))
+const DEFAULT_GID = DEFAULT_SELECTOR
```

Comment:
Is it possible to remove `DEFAULT_GID`, since it's an alias to `DEFAULT_SELECTOR ` anyway?